src/efs_parser/BaseFile.py

```diff
@@ -38,25 +41,30 @@ class BaseFile(object):
 
     def __init__(self, file):
         self.file = file
-        self.local = self.is_local(file)
+        self.file_source = self.get_file_source(file)
         self.endian = "="
         self.compressed = True
         self.conn = None
         self.stats = {
             "iotime"
         }
         self.byteRanges = {}
+        self.bucketname="encode-public"
+
```

Comment:
this shouldn't be here, the idea is to be able to work on any public s3 bucket not only for this

Reply:
Done